README.md

```diff
@@ -70,11 +56,10 @@ See [the list of releases][6] to find out about feature changes.
 
 [24]: https://groups.google.com/forum/#!forum/projectvelero
 [25]: https://kubernetes.slack.com/messages/velero
-[26]: https://velero.io/docs/zenhub
 
-[28]: https://velero.io/docs/install-overview
-[29]: https://velero.io/docs/
-[30]: https://velero.io/docs/troubleshooting
+[28]: install-overview.md
```

Comment:
Need to revert these link changes (28-100) - they still need to point to `velero.io/...`